app/views/contexts/hub_context.py

```diff
@@ -67,6 +67,7 @@ def __init__(
         self._schema = schema
         self._survey_complete = survey_complete
         self._placeholder_renderer = None
+        self._path_finder = path_finder
```

Comment:
I don't think we should add path_finder as a dependency to this, we could just pass the enabled section ids.